src/components/FruitFieldset.jsx

```diff
@@ -0,0 +1,19 @@
+import {useState} from "react";
+
+function FruitFieldset({fruitName, fruitValue, handleFruitChange}) {
+
+    // const [fruitValue, setFruitValue] = useState(0)
+
+    return (
+        <fieldset className="fruit_card">
+            {fruitName}
+            <div className="fruit_counter_set">
+                <button onClick={() => handleFruitChange(fruitValue - 1 >= 0 ? fruitValue - 1 : fruitValue)}> -</button>
```

Comment:
Met het `disabled` attribuut kun je ervoor zorgen dat de onClick niet eens uitgevoerd wordt. Dat is netter dan het binnen de callback goed te regelen